#!/bin/sh /usr/share/dpatch/dpatch-run
## 209-magic-update-jpeg.dpatch by Mathieu Malaterre <mathieu.malaterre@gmail.com>
##
## DP: Extends JPEG 2000 magic (Closes: #439005).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/jpeg file-4.21/magic/Magdir/jpeg
--- file-4.21.orig/magic/Magdir/jpeg	2004-07-24 20:38:55.000000000 +0000
+++ file-4.21/magic/Magdir/jpeg	2007-08-26 08:12:23.000000000 +0000
@@ -153,3 +153,8 @@
 
 # From: David Santinoli <david@santinoli.com>
 0	string		\x00\x00\x00\x0C\x6A\x50\x20\x20\x0D\x0A\x87\x0A	JPEG 2000 image data
+
+# Type: JPEG 2000 codesream
+# From: Mathieu Malaterre <mathieu.malaterre@gmail.com>
+0	belong		0xff4fff51						JPEG 2000 codestream
+45	beshort		0xff52
